components/mastodon/actions/post-status/post-status.mjs

```diff
@@ -4,63 +4,110 @@ import { VISIBILITY_OPTIONS } from "../../common/constants.mjs";
 export default {
   key: "mastodon-post-status",
   name: "Post Status",
-  description: "Publish a status with the given parameters. [See the docs here](https://docs.joinmastodon.org/methods/statuses/#create)",
-  version: "0.0.1",
+  description: "Publish a status with the given parameters. [See the documentation](https://docs.joinmastodon.org/methods/statuses/#create)",
+  version: "0.0.2",
   type: "action",
   props: {
     mastodon,
     status: {
       type: "string",
       label: "Status",
-      description: "The text content of the status",
+      description: "The text content of the status.",
     },
     inReplyToId: {
       type: "string",
       label: "In Reply To ID",
-      description: "ID of the status being replied to, if status is a reply",
+      description: "ID of the status being replied to, if the status is a reply.",
       optional: true,
     },
     sensitive: {
       type: "boolean",
       label: "Sensitive",
-      description: "Mark status and attached media as sensitive? Defaults to false",
+      description: "Mark the status and attached media as sensitive? Defaults to false.",
       optional: true,
       default: false,
     },
     spoilerText: {
       type: "string",
       label: "Spoiler Text",
-      description: "Text to be shown as a warning or subject before the actual content. Statuses are generally collapsed behind this field",
+      description: "Text to be shown as a warning or subject before the actual content. Statuses are generally collapsed behind this field.",
       optional: true,
     },
     visibility: {
       type: "string",
       label: "Visibility",
-      description: "Sets the visibility of the posted status to `public`, `unlisted`, `private`, or `direct`",
+      description: "Set the visibility of the posted status to `public`, `unlisted`, `private`, or `direct`.",
       options: VISIBILITY_OPTIONS,
       optional: true,
     },
     scheduledAt: {
       type: "string",
       label: "Scheduled At",
-      description: "ISO 8601 Datetime at which to schedule a status. Must be at least 5 minutes in the future.",
+      description: "ISO 8601 DateTime at which to schedule a status. Must be at least 5 minutes in the future.",
       optional: true,
     },
+    shouldSplit: {
+      type: "boolean",
+      label: "Split to multiple messages",
+      description: "If the status content is longer than 500 characters, it will be split, respecting words, and posted to the subsequent thread.\n\nThis action will return the array of submitted posts.",
```

Comment:
```suggestion
      description: "If the status content is longer than 500 characters, it will be split, respecting words, and posted to the subsequent thread.\n\nThis action will return the array of submitted posts.";
```

Reply:
Removed extra comma at the end of the sentence.